src/bipartite.jl

```diff
@@ -0,0 +1,96 @@
+"""
+    BipartiteView{T<:Real} <: AbstractGraph{Int64}
+
+
+Creates a bipartite view of a hypergraph.
+Note this is a view - changes to the original hypergraph will be automatically reflected in the view.
+
+**Constructors**
+
+    BipartiteView{T}(h::Hypergraph{T}) where {T<:Real}
+
+Construct a bipartite view of a hypergraph for processing with LightGraphs
+
+
+**Arguments**
+
+* `h` : a hypergraph
+
+"""
+struct BipartiteView{T<:Real} <: AbstractGraph{Int64}
+    v2he::Vector{Dict{Int,T}}
```

Comment:
why not simply hold a `Hypergraph`. This breaks encapsulation and will be harder to maintain in the long range.